scripts/components/OpenSearch/build.sh

```diff
@@ -137,10 +140,11 @@ echo "Building OpenSearch for $PLATFORM-$DISTRIBUTION-$ARCHITECTURE"
 ./gradlew :distribution:$TYPE:$TARGET:assemble -Dbuild.snapshot=$SNAPSHOT
 
 # Copy artifact to dist folder in bundle build output
-[[ "$SNAPSHOT" == "true" ]] && IDENTIFIER="-SNAPSHOT"
 ARTIFACT_BUILD_NAME=`ls distribution/$TYPE/$TARGET/build/distributions/ | grep "opensearch-min.*$QUALIFIER.$EXT"`
+ARTIFACT_TARGET_BUILD_NAME=opensearch-min-$VERSION$IDENTIFIER-$QUALIFIER.$EXT
+[ ! -z "$QUALIFIER_ALT" ] && ARTIFACT_TARGET_BUILD_NAME=opensearch-min-$VERSION$IDENTIFIER-$QUALIFIER_ALT.$EXT
```

Comment:
For the right naming convention a change should be made in OpenSearch rather than renaming the artifact here. It can be confusing if someone tries to build OpenSearch rpm using core repo.
I think this issue was created for the same purpose: https://github.com/opensearch-project/opensearch-build/issues/397

Reply:
Not knowing this issue exist to start with.
The reason I havent change core is due to yum probably need us to keep x86_64 naming.
Will keep this PR open for a while.

Reply:
We have the naming as a requirements here, the reason I rename it is to keep consistency:
https://github.com/opensearch-project/opensearch-build/issues/1548